src/genie/libs/parser/iosxe/show_wireless.py

```diff
@@ -0,0 +1,229 @@
+import re
```

Comment:
Write all parser commands on top as comment

Reply:
Can you clarify what you mean? Do you mean this comment block? 
```
# =================================
# Schema for:
#  * 'show wireless client summary'
# =================================
```

Reply:
yes